server/migration/1616699371961-UserSettingsRemoveEnableNotifications.ts

```diff
@@ -0,0 +1,32 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
This file needs to be regenerated since we have had another migration in the meantime.